great-idea-website/css/index.css

```diff
@@ -63,4 +63,169 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+nav {
+	width: 600px;
+	height:30px;
+	box-sizing:border-box;
+	margin-left:100px;
+	word-spacing: 40px;
+	color:grey;
+	margin-bottom:40px;
+	font-size:15px;
+	margin-top:50px;
+}
+	
+.logo {
+	width: 200px;
+	height:30px;
```

Comment:
Be careful of arbitrarily changing the size of the image. Since these sizes don't quite match up with the original aspect ratio of the image, the image will appear slightly distorted.

If you need to change the size of one dimension, but don't care about the other, you can do something like this:
```
.logo {
  width: 200px;
  height: auto; /* or vice versa */
}
```